tests/validation/tests/v3_api/common.py

```diff
@@ -23,6 +24,9 @@
 MACHINE_TIMEOUT = float(os.environ.get('RANCHER_MACHINE_TIMEOUT', "1200"))
 
 TEST_IMAGE = os.environ.get('RANCHER_TEST_IMAGE', "sangeetha/mytestcontainer")
+TEST_UPGRADE_IMAGE_1 = os.environ.get('RANCHER_TEST_UPGRADE_IMAGE_1', "nginx")
+TEST_UPGRADE_IMAGE_2 = os.environ.get('RANCHER_TEST_UPGRADE_IMAGE_2', "ubuntu")
```

Comment:
Can we use `TEST_UPGRADE_NGINX` and `TEST_UPGRADE_UBUNTU` instead of 1 and 2 for the name? It is confusing for the variable name.

Reply:
We could confirm `TEST_UPGRADE_IMAGE_1` and `TEST_UPGRADE_IMAGE_2` to any image, on the other hand, Window could not deploy `ubuntu` image.

Reply:
```
if TEST_OS == "windows":
    DEFAULT_TIMEOUT = 300
    # Windows base image needs to match the host kernel release id(1809, 1903, ...),
    # We need to use the form of the docker manifest to package the image below.
    TEST_IMAGE_NGINX = "maiwj/nginx:1.16.0-windows-1809"
    TEST_IMAGE_OS_BASE = "mcr.microsoft.com/windows/servercore:1809"
```
Judge images by `TEST_OS`